docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:xenial
+ 
+RUN apt-get update && apt-get install -y curl lsb-release
```

Comment:
Also let's move each package into it's own line (as well as the `rm -rf` command)